tests/baselines/reference/user/webpack.log

```diff
@@ -5,6 +5,9 @@ lib/ContextModule.js(465,34): error TS2341: Property 'moduleGraph' is private an
 lib/ContextModule.js(767,34): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
 lib/Dependency.js(139,29): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
 lib/ExternalModule.js(284,54): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
+lib/FileSystemInfo.js(357,4): error TS2345: Argument of type '({ type, context, path, expected }: { type: any; context: any; path: any; expected: any; }, callback: any) => void' is not assignable to parameter of type 'AsyncFunction<{ type: number; path: string; context?: string; expected?: string; }, Error>'.
```

Comment:
@ahejlsberg this one's new to your PR.

Reply:
Yes, it's new and it appears to be correct. According to the type annotations, the `queue` method expects an argument of type `AsyncFunction<T, E>` which is equivalent to `(callback: (err?: E, result?: T) => void) => void`. But the arrow function provided expects an object in the first parameter. Pretty sure the type annotations are wrong somehow, but certainly we're correct to report an error.